locales/bn/bn.json

```diff
@@ -429,26 +429,26 @@
     "Peru": "পেরু",
     "Philippines": "ফিলিপাইন",
     "Photo": "ছবি",
-    "Pitcairn": "মৃত্যুর স্থান",
+    "Pitcairn": "পিটকেয়ার্ন",
     "Please accept the terms of service.": "অনুগ্রহ করে সেবার শর্তসমূহ গ্রহণ করুন",
     "Please click the button below to verify your email address.": "আপনার ইমেল ঠিকানা যাচাই করার জন্য নিচের বাটনে ক্লিক করুন.",
     "Please confirm access to your account by entering one of your emergency recovery codes.": "আপনার জরুরী পুনরুদ্ধারের কোড এক লিখে আপনার অ্যাকাউন্টে অ্যাক্সেস নিশ্চিত করুন.",
     "Please confirm access to your account by entering the authentication code provided by your authenticator application.": "আপনার প্রমাণকারী অ্যাপ্লিকেশনের দ্বারা প্রদত্ত প্রমাণীকরণ কোড লিখে আপনার অ্যাকাউন্টে অ্যাক্সেস নিশ্চিত করুন.",
     "Please confirm your password before continuing.": "Please confirm your password before continuing.",
     "Please copy your new API token. For your security, it won't be shown again.": "আপনার নতুন এপিআই টোকেন কপি করুন. আপনার নিরাপত্তার জন্য, এটা আবার দেখানো হবে না.",
```

Comment:
Is the correct character used to end a sentence?

In other places, I noticed the use of the vertical slash character (`|`), but here - a period (`.`).

This translation was by machine.